app/controllers/flow_controller.rb

```diff
@@ -45,7 +55,16 @@ def index
   end
 
   def about
-    'xx'
+  end
+
+  def restrictions
```

Comment:
If I read this correctly, this could return a lot of records - we avoid any code that pulls a long list of records into memory. I think given above note on `status` - this goes away